bin/import-wp-css-storybook.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+STORYBOOK_WORDPRESS_DIR="$DIR/../storybook/wordpress";
+STORY_BOOK_CSS_PATH="$DIR/../storybook/wordpress/css";
+TMP_DIR="$DIR/../storybook/wordpress/tmp";
+ARCHIVE_CSS_PATH="wordpress/wp-admin/css";
```

Comment:
I think these paths may need to be relative.  The directory that gets added to my `woocommerce-admin` folder looks like its relative from the root directory.

In my environment:
`woocommerce-admin/Sites/my-store/app/public/wp-content/plugins/woocommerce-admin/storybook/wordpress/css/...`

<img width="1017" alt="Screen Shot 2020-10-07 at 10 08 26 AM" src="https://user-images.githubusercontent.com/10561050/95342378-0bd23280-08c0-11eb-9c80-1b33b269fee9.png">
<img width="1005" alt="Screen Shot 2020-10-07 at 10 01 41 AM" src="https://user-images.githubusercontent.com/10561050/95342380-0bd23280-08c0-11eb-8976-1385c671f001.png">

Reply:
@joshuatf Could you clarify what your setup is?

It is a relative path to the location of the bash script. I'm trying to work out how that could be a problem if you just have `woocommerce-admin` cloned normally? I can run this script from absolutely any subdirectory in the repo right now and it will correctly place the wordpress assets inside the storybook folder. I can even run it from outside the repo and it places it correctly.

Some ideas I can think of:

1. I haven't tested this on Linux, are you on Linux?
2. Maybe the script's mkdir -p failed with some kind of permission issue due to RW permissions? Could you try `sudo` on these lines: 
```
mkdir -p $STORY_BOOK_CSS_PATH;
mkdir -p $TMP_DIR;
```

Reply:
1. No, I'm on OSX.
2. Ran `npm run storybook` from the `woocommerce-admin` dir.
3. I did not notice any, but I will check again and see if I can find any.

The folder is created, it's just created in a deeply nested structure of my entire path to `woocommerce-admin` so my gut tells me this is not permissions related, but I'll test around this today and see if I can figure out what's happening.

Reply:
@joshuatf If you clone it into a fresh directory does it work?

I'd also be happy to try help debug or at least try get to the bottom of why this script is not working in some situations.

Looking at the rsync output where it can't find the directory it looks like in your environment your woocommerce-admin dir is at:

`/Users/joshua/Local Sites/my-store/app/public/wp-content/plugins/woocommerce-admin` Is that correct? If thats correct then the path that `rsync` is looking for is exactly where I'd expect it to place the tmp dir. it just creates the tmp dir inside the storybook folder.

Sorry I can't be more help!

Reply:
Just ran into the same issue Josh was here; I wonder if it has to do that we're both using Flywheel for our local setup? It's creating another directory at the plugin root:

![image](https://user-images.githubusercontent.com/444632/117219533-52d2c880-adba-11eb-9436-e147c445e8cc.png)

Poking at it now, but if anyone has any suggestions I'm all 👂🏻 !
cc @samueljseay @joshuatf

Reply:
Ah, may have figured it out. It fails when there is a space in the path since it's missing quotes (I _think_)

Reply:
@joelclimbsthings yup I fixed this in a recent PR

Reply:
https://github.com/woocommerce/woocommerce-admin/pull/6934

Reply:
😆 . Just noticed that when I tried to merge my own fix 😭 . Thanks @samueljseay